app/src/main/res/values-el/strings.xml

```diff
@@ -21,21 +21,21 @@
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- App title has to have less than 50 characters. If you cannot squeeze it, just remove a part of it -->
-    <string name="app_title">Simple App Launcher - Launch apps easily &amp; quickly</string>
+    <string name="app_title">Απλός Εκκινητής - Εύκολη &amp; γρήγορη εκκίνηση εφαρμογών</string>
```

Comment:
app title has to be max 50 chars long, even if "&amp;" counts at one. It currently is 53 chars long